### PR TITLE
Remove some of the Novoda marketing!

### PR DESCRIPTION
**Pull request by charroch**
I don't think there is a need to market Novoda here. We should focus on specifying how the PR should be constructed and what and what not will be accepted.

We should also add licensing info but that will come later.

### CODE REVIEW

**Comment by blundell** [comment]
:+1:

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -19,10 +17,12 @@ The PR should follow this format:
 
 ## Writing tests
 
-Novoda is known for delivering quality, and one key point to create high quality software is by complementing it with a meaningful suite of tests. When you issue a PR, please take some time to consider writing tests for the issue. For example if you're solving a bug, you could write the test that reproduces the bug first, then fix the issue. This makes sure the bug doesn't come back later.
+When you issue a PR, please take some time to consider writing tests for the issue. For example if you're solving a bug, you could write the test that reproduces the bug first, then fix the issue. This makes sure the bug doesn't come back later.
 
 Also make sure the project builds and all the tests pass before creating the PR.
 
+A non tested PR will not be merged back.
```
Comment (ouchadam) [inline CONTRIBUTING.md]: non tested? unit/manual?

Comment (danybony) [inline CONTRIBUTING.md]: Yes, would be worth explain a bit this.
A part from this :+1:

**Comment by zegnus** [comment]
Are we happy with this so we can merge?

**Comment by ouchadam** [comment]
haven't had a response yet

File: CONTRIBUTING.md (hunk 1 above)
Comment (charroch) [inline CONTRIBUTING.md]: I am happy with keeping it open ended.

**Comment by charroch** [comment]
I am happy with the way it is is. Up @ouchadam and @danybony to :+1: it

**Comment by ouchadam** [comment]
:+1: fine with me

**Comment by danybony** [comment]
Good for me as well :+1:

**Pull request merged**